src/live-programming.rst

```diff
@@ -20,9 +20,203 @@ Data Analysis Environments
 Debugging
 =========
 
+  "The EDSAC was on the top floor of the building and the tape-punching
+  and editing equipment one floor below... It was on one of my journeys
+  between the EDSAC room and the punching equipment that...the realization
+  came over me with full force that a good part of the remainder of my
+  life was going to be spent in finding errors in my own programs."
+
+  -- *Memoirs of a Computer Pioneer*, Maurice Wilkes
+
+Much of the activity of programming involves **debugging**, i.e.,
+diagnosing and preventing undesired program behavior.
+Wilkes's early realization of this fact was echoed and amplified in a 2002
```

Comment:
Results from publications need to be cited. This should be a bib-item, either directly inline here or at the end of this section with a citation.

Reply:
still need this